junit-platform-gradle-plugin/src/main/groovy/org/junit/platform/gradle/plugin/EnginesExtension.groovy

```diff
@@ -39,5 +39,4 @@ class EnginesExtension {
 	void exclude(String... engineIds) {
 		this.exclude.addAll engineIds
 	}
-
```

Comment:
:+1: 
GrEclipse is not configurable in that respect and insists (even with the latest snapshot) on that formatting. I used _spotless_ `replaceRegex` as a work-around.